op-test-sequencer/sequencer/backend/work/builders/standardbuilder/config.go

```diff
@@ -0,0 +1,109 @@
+package standardbuilder
+
+import (
+	"context"
+
+	"github.com/ethereum-optimism/optimism/op-node/rollup"
+	"github.com/ethereum-optimism/optimism/op-node/rollup/derive"
+	"github.com/ethereum-optimism/optimism/op-service/client"
+	"github.com/ethereum-optimism/optimism/op-service/endpoint"
+	"github.com/ethereum-optimism/optimism/op-service/retry"
+	"github.com/ethereum-optimism/optimism/op-service/sources"
+	"github.com/ethereum-optimism/optimism/op-test-sequencer/sequencer/backend/work"
+	"github.com/ethereum-optimism/optimism/op-test-sequencer/sequencer/seqtypes"
+)
+
+type Config struct {
+	// L1 execution-layer RPC endpoint
+	L1EL endpoint.MustRPC `yaml:"l1EL,omitempty"`
+
+	// L2 execution-layer RPC endpoint
+	L2EL endpoint.MustRPC `yaml:"l2EL,omitempty"`
+	// L2 consensus-layer RPC endpoint
+	L2CL endpoint.MustRPC `yaml:"l2CL,omitempty"`
+}
+
+type CloseFn func()
+
+func (fn *CloseFn) Prepend(other func()) {
+	self := *fn
+	*fn = func() {
+		other()
+		self()
+	}
+}
+
+func (fn CloseFn) Maybe() (cancel func(), close func()) {
+	do := true
+	cancel = func() {
+		do = false
+	}
+	close = func() {
+		if do {
+			fn()
+		}
+	}
+	return
+}
+
+func (c *Config) Start(ctx context.Context, id seqtypes.BuilderID, opts *work.ServiceOpts) (work.Builder, error) {
```

Comment:
I think this constructor should be within `builder.go`, and probably not a method of `c *Config`.

In any case we can refactor in upcoming PRs related to `op-test-sequencer`